Source/IGListAdapterUpdater.m

```diff
@@ -155,7 +153,8 @@ - (void)performBatchUpdatesWithCollectionView:(UICollectionView *)collectionView
 
     // if the collection view isn't in a visible window, skip diffing and batch updating. execute all transition blocks,
     // reload data, execute completion blocks, and get outta here
-    if (_canBackgroundReload && collectionView.window == nil) {
+    BOOL iOS83OrLater = (NSFoundationVersionNumber >= NSFoundationVersionNumber_iOS_8_3);
+    if (iOS83OrLater && _skipsDiffingWhenOffscreen && collectionView.window == nil) {
```

Comment:
Let's use the getter `self.skipsDiffingWhenOffscreen` since we're using more than an ivar